samples/RateLimiter/Program.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore.RateLimiting;
+using Ocelot.DependencyInjection;
+using Ocelot.Middleware;
+using System.Threading.RateLimiting;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Configuration.AddJsonFile("ocelot.json");
+builder.Services.AddOcelot();
+
+builder.Services.AddRateLimiter(op =>
```

Comment:
In config file, RateLimiterPolicyName must be set when using dotnet rate limiter. This policy must be configured at startup. Dotnet allows registering multiple policies. Each endpoint defines which policy to use. See [here](https://learn.microsoft.com/en-us/aspnet/core/performance/rate-limit?view=aspnetcore-8.0#fixed-window-limiter)
It is very similar to how to Authentication is setup with Ocelot.

Reply:
```json
      "RateLimitOptions": {
        "EnableRateLimiting": true,
        "RateLimitMiddlewareType": "DotNet",
        "RateLimitPolicyName": "fixed"
      }
```
OK [I see now](https://github.com/ThreeMammals/Ocelot/pull/2188/files/617c85aafd1de808c91673d6f72c640dfb648e33#diff-cb4d86798213d830f3e239d6ff020de91ca1de6807dec9bcb77ca8a2e41789ffR28-R32)!

> It is very similar to how to Authentication is setup with Ocelot.

Yeap! I am familiar with the Auth policy concept.

Reply:
The concern is that the property names are overly long with unnecessary prefixes. My argument is that properties within `RateLimitOptions` should not include the `RateLimit` prefix, suffix, or be embedded within the name.

I suggest to rename the properties to more simplified names as shown below:
```diff
-        "RateLimitMiddlewareType": "DotNet",
-        "RateLimitPolicyName": "fixed"
+        "MiddlewareType": "DotNet",
+        "PolicyName": "fixed"
```
Alternatively, it may be even more concise to shorten them as follows:
```diff
-        "RateLimitMiddlewareType": "DotNet",
-        "RateLimitPolicyName": "fixed"
+        "Middleware": "DotNet",
+        "Policy": "fixed"
```
Which version do you prefer?

Reply:
Sure. I prefer the second option.

Reply:
renamed as discussed.